### PR TITLE
format services docs

### PR DESCRIPTION
**Pull request by butlerx**
what isnt in this is in #56

### CODE REVIEW

File: docs/services/dovecot.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ These docs are incomplete, but I've tried to update them with what I can
 remember.
 
 ### imaps
```
Comment (VoyTechnology) [inline docs/services/dovecot.md]: Unrelated: IMAPs?

File: docs/services/dovecot.md [hunk 2]
```diff
@@ -9,9 +9,11 @@ These docs are incomplete, but I've tried to update them with what I can
...
+
 We only use imaps, on port 993. This is set in dovecot.conf
 
 ### ssl
```
Comment (VoyTechnology) [inline docs/services/dovecot.md]: Unrelated: SSL?

File: docs/services/dovecot.md [hunk 3]
```diff
@@ -20,15 +22,17 @@ copy in `/etc/dovecot`.
...
+```
 
-
-[root@morpheus /]# ls -ld /var/mail/indexes
+``` bash
+$ ls -ld /var/mail/indexes
 drwxrwsrwt 58 root mail 4.0K 2009-07-28 14:11 /var/mail/indexes
 ```
 
 ### ldap
```
Comment (VoyTechnology) [inline docs/services/dovecot.md]: Unrelated: LDAP

File: docs/services/nfs.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
 # Current NFS setup
 
-**Please try to keep this up-to-date!**
+Do **PLEASE try to keep this up-to-date!**
```
Comment (VoyTechnology) [inline docs/services/nfs.md]: This `Do` makes no sense here

Comment (VoyTechnology) [inline docs/services/nfs.md]: `Do` makes no sense here

**Review by VoyTechnology — CHANGES_REQUESTED** [review]

File: docs/services/nfs.md (hunk 4 above)
Comment (butlerx) [reply docs/services/nfs.md]: do please is correct grammer

**Review by VoyTechnology — APPROVED** [review]
LGTM

**Pull request merged**